src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  personas: Person[],
+  selectedPersonId: string,
+};
+
+export const PersonList: React.FC<Props> = ({
+  personas,
+  selectedPersonId,
+}) => {
+  const isSelected = (person: Person) => person.slug === selectedPersonId;
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {personas.map(person => {
+          const mother = personas.find(mom => mom.name === person.motherName);
+          const father = personas.find(dad => dad.name === person.fatherName);
```

Comment:
what is `personas`?
will be better to add `mother` and `father` properties for each person after retrieving them from server
instead of searching objects on each callback call

Reply:
sorry, naming is 80 level... my bad)